src/components/pip/pip-parent.tsx

```diff
@@ -46,11 +46,6 @@ export class PipParent extends Component<PIPParentComponentProps> {
       }
     }
 
-    const videoContainerStyles = {
-      height: `${props.playerHeight + 'px'}`,
-      width: `${props.playerWidth + 'px'}`
-    };
-
-    return <div className={styleClass.join(' ')} style={videoContainerStyles} ref={this.videoContainerRef} />;
+    return <div className={styleClass.join(' ')}  ref={this.videoContainerRef} />;
```

Comment:
@JonathanTGold removing height for playerContainer will cause the dual to stretch on the main player. see screenshot-
![image](https://user-images.githubusercontent.com/79077248/212690697-a8f1bf3a-3588-4bfa-ae4b-9137192c11ae.png)
you have to keep the height.